gammapy/modeling/models/core.py

```diff
@@ -428,6 +430,18 @@ def read(cls, filename):
     def from_yaml(cls, yaml_str, path=""):
         """Create from YAML string."""
         data = yaml.safe_load(yaml_str)
+        if "metadata" in data.keys():
+            creator_version = data["metadata"]["creator"].split()[1]
+            if check_version(creator_version) < 0:
+                log.warning(
+                    f"The currently used Gammapy version is older than the one used to create the model ({creator_version})"
+                )
+            else:
+                log.debug(
+                    f"The currently used Gammapy version is compatible with the one used to create the model ({creator_version})"
```

Comment:
Is it better to say "compatible" or just "same". Older versions are not necessarily incompatible

Reply:
This also includes the case of check_version > 1 ? Older versions may not support new models

Reply:
They can never support them! except if the code is changed... But as you can see, this test gives only a log message. There is no stop-over

Reply:
> Is it better to say "compatible" or just "same". Older versions are not necessarily incompatible

I do not understand your point... because the code does not say that here